glibc/pkg.yaml

```diff
@@ -0,0 +1,45 @@
+name: glibc
+variant: debian
+install:
+  - make
+  - gawk
+  - bison
+  - python3
+dependencies:
+  - stage: bootstrap-gcc
```

Comment:
this is something I could not figure out, if we depend on `bootstrap` which includes `bootstrap-libstdcxx` glibc build barfs with error saying it cannot find `-lgcc_s`. I couldn't figure out why, so just depend on `bootstrap-gcc` which does not include `bootstrap-libstdcxx`